library/UnicornHat/unicornhat.py

```diff
@@ -111,6 +115,16 @@ def rotation(r=0):
         raise ValueError('Rotation must be 0, 90, 180 or 270 degrees')
 
 
+def get_rotation():
+    """Get the display rotation value
+
+    Returns an integer, either 0, 90, 180 or 270
+    """
+    global _requested_rotation
```

Comment:
No need to declare `global _requested_rotation` here, since the value is being read and not assigned to, and will thus find the instance in the global scope implicitly. If you'd use `_requested_rotation = some_value` then it would be required to stop the creation of a new variable in the `get_rotation` scope that masks the one in the global scope.